autosklearn/util/__init__.py

```diff
@@ -1,6 +1,6 @@
 # -*- encoding: utf-8 -*-
-from .common import *
```

Comment:
Would it be possible to change/remove these imports?

Reply:
yes

Reply:
No, wait. Just to clarify. Do you mean change/remove without modifying the way those functions are imported by the rest of the package?

Reply:
No, I meant removing those imports and changing the way other modules import the utils.

Reply:
Alright then!